src/Development/Shake/Internal/Options.hs

```diff
@@ -301,14 +306,24 @@ tyHidden = mkDataType "Development.Shake.Types.Hidden" []
 
 -- | The verbosity data type, used by 'shakeVerbosity'.
 data Verbosity
-    = Silent -- ^ Don't print any messages.
-    | Quiet  -- ^ Only print essential messages, typically errors.
-    | Normal -- ^ Print errors and @# /command-name/ (for /file-name/)@ when running a 'Development.Shake.traced' command.
-    | Loud   -- ^ Print errors and full command lines when running a 'Development.Shake.command' or 'Development.Shake.cmd' command.
-    | Chatty -- ^ Print errors, full command line and status messages when starting a rule.
+    = Silent  -- ^ Don't print any messages.
+    | Err     -- ^ Only print error messages.
+    | Warn    -- ^ Print errors and warnings.
+    | Info    -- ^ Print errors, warnings and @# /command-name/ (for /file-name/)@ when running a 'Development.Shake.traced' command.
+    | Debug   -- ^ Print errors, warnings and full command lines when running a 'Development.Shake.command' or 'Development.Shake.cmd' command.
+    | Verbose -- ^ Print errors, warnings, full command line and status messages when starting a rule.
     | Diagnostic -- ^ Print messages for virtually everything (mostly for debugging).
       deriving (Eq,Ord,Show,Read,Typeable,Data,Enum,Bounded)
 
+-- | A bidirectional pattern synonym for 'Err'.
+pattern Quiet  = Err
```

Comment:
I'd move these into `Development.Shake` as these are effectively on the deprecation path now.

Reply:
From https://gitlab.haskell.org/ghc/ghc/wikis/pattern-synonyms/associating-synonyms I had understood that defining them here keeps them bundled with `Verbosity` so, for recent enough versions of GHC, if you import `Verbosity(..)` you get the old patterns and constructors too.

Maybe that isn't true but, either way, it's probably unnecessary, so I moved them as you suggested.